column_encoder.py

```diff
@@ -272,23 +273,39 @@ def transform(self, X):
 class MinHashEncoder(BaseEstimator, TransformerMixin):
     """
     minhash method applied to ngram decomposition of strings
+
+    Parameters
+    ----------
+    n_components : integer
+        The number of dimension for each sample
+    ngram_range : tuple (min_n, max_n)
+        The lower and upper boundary of the range of n-values for different
+        n-grams to be extracted. All values of n such that min_n <= n <= max_n
+        will be used.
+    hashing : {'fast_hash', 'murmur_hash'}
```

Comment:
Maybe rename to "murmur" and "fast" the acceptable values the "_hash" is redundant info.